pkg/mount/deprecated.go

```diff
@@ -39,15 +44,18 @@ const (
 	STRICTATIME = sysmount.STRICTATIME
 )
 
+// Deprecated: use github.com/moby/sys/mount instead.
 //nolint:golint
```

Comment:
Is the linter complaining without the `//nolint` comments? The downside of these is that they are only taken into account by our custom linters (such as golang-ci-lint), but, e.g. ignored by `golang.org/x/lint` itself, and thus would show up on https://goreportcard.com/report/github.com/containerd/containerd#golint

Reply:
nolint:golint was added to get rid of "not documented" warnings. Maybe when "Deprecated" documentation is added, this is no longer needed. Let me try.

Reply:
Let's see if removing `golint:nolint` works (if not, I'll revert)

Reply:
```
[2020-03-19T18:27:31.851Z] // Deprecated: use github.com/moby/sys/mount instead.
[2020-03-19T18:27:31.851Z] ^
[2020-03-19T18:27:31.851Z] pkg/mount/deprecated.go:48:1: comment on exported type FilterFunc should be of the form "FilterFunc ..." (with optional leading article) (golint)
[2020-03-19T18:27:31.851Z] // Deprecated: use github.com/moby/sys/mountinfo instead.
[2020-03-19T18:27:31.851Z] ^
```

Nope, `golint` still complains, reverting the last change back